indico/modules/users/models/users.py

```diff
@@ -131,7 +132,10 @@ def display_full_name(self):
 
 
 def format_display_full_name(user, obj):
-    name_format = user.settings.get('name_format') if user else NameFormat.first_last
+    from indico.modules.events.layout import layout_settings
+    name_format = layout_settings.get(g.rh.event, 'name_format') if 'rh' in g else NameFormat.inherit
+    if name_format is NameFormat.inherit:
```

Comment:
don't use `is` here

Reply:
This is useful if you're not sure whether you should use `is` or `==`:
https://stackoverflow.com/questions/132988/is-there-a-difference-between-and-is